label/label.c

```diff
@@ -78,7 +78,7 @@ int main(int argc, char *argv[])
 	if (version_only)
 		exit(EXIT_FAILURE);
 
-	if (argc - optind != 1)
+	if (argc - optind < 1)
```

Comment:
It could allow more arguments. how about this?
And could you change the prefix of a title from 'exfatprogs' to exfatlabe?
```
diff --git a/label/label.c b/label/label.c
index 12d027b..f7e12d9 100644
--- a/label/label.c
+++ b/label/label.c
@@ -41,7 +41,7 @@ int main(int argc, char *argv[])
        struct exfat_user_input ui;
        bool version_only = false;
        int serial_mode = 0;
-       int flags = 0;
+       int flags = -1;
        unsigned long volume_serial;
 
        init_user_input(&ui);
@@ -78,7 +78,7 @@ int main(int argc, char *argv[])
        if (version_only)
                exit(EXIT_FAILURE);
 
-       if (argc - optind != 1)
+       if (flags == -1)
                usage();
 
        ui.dev_name = argv[serial_mode + 1];
```

Reply:
This check is only not applicable to set label.
How about this?

```diff
- if (argc - optind != 1) 
+ if (argc - optind != 1 && flags != EXFAT_SET_VOLUME_LABEL) 
```

Reply:
@YuezhangMo You need to consider EXFAT_SET_VOLUME_SERIAL. So it should be :
+       if (argc - optind != 1 && flags != EXFAT_SET_VOLUME_LABEL &&
+           flags != EXFAT_SET_VOLUME_SERIAL)

Reply:
Yes, you are right.

Reply:
@YuezhangMo Thanks for your check and suggestion:)
@RomainNaour Can you update your patch if you agree about this change ?

Reply:
Hello,
Thanks for your review.
I pushed an updated version.
Feel free to amend the patch :)
Thanks!